namui-cli/electron/src/main.js

```diff
@@ -71,7 +71,7 @@ function getConfig() {
     const config = {
         test: false,
         port: 8000,
-        resourceRoot: path.join(__dirname, "../.."),
+        applicationRoot: path.join(__dirname, "../../.."),
```

Comment:
name `resourceRoot` was too ambiguous so I changed it to applicationRoot. I think you want to point `resources` directory for `{applicationRoot}/resources/app/src/main.js`, then it should be resource's'Root.